server/configurations/local.yaml

```diff
@@ -303,7 +303,8 @@ internal:
     #     local-domain-suffix: "@example.org"
     #     local-domain-value: 123456
     # List of user IDs that can use the admin API endpoints.
-    admins: []
+    admins:
```

Comment:
undo? you can create a `museum.yaml` file in server/ directory and add your local configurations in that.

Reply:
Done in the new commit